lib/msgfmt/testdata/format/claude/remove-task-tool-call/msg.txt

```diff
@@ -1,5 +1,13 @@
 > Build a snake game
 
+● coder - coder_report_task (MCP)(summary: "Snake game created
+                                 successfully at snake-game.html",
+                                 link: "file:///home/coder/snake-ga
+                                 me.html", state: "working")
+  ⎿  {
+       "message": "Thanks for reporting!"
+     }
+
```

Comment:
On its own, adding this does not cause any test failures on `main`. We should have a sample input that reproduces the issue so that we can have confidence in the fix.